foreman_expire_hosts.gemspec

```diff
@@ -18,9 +18,9 @@ Gem::Specification.new do |s|
 
   s.require_paths = ['lib']
 
-  s.add_dependency 'deface'
   s.add_dependency 'bootstrap-datepicker-rails'
+  s.add_dependency 'deface'
 
-  s.add_development_dependency 'rubocop', '0.49.1'
   s.add_development_dependency 'rdoc'
+  s.add_development_dependency 'rubocop', '0.51.0'
```

Comment:
one of the most stupid cop ever, just after you need to have space around "TODO" in comments :-)